### ISSUE TITLE
Question: uk-width-*-10

### ISSUE THREAD

**Issue by kiarashi** [uikit/uikit#2593]
Would it be possibe to put back the 10 columns inside UIKit3?

I use uk-width-3-10 and uk-width-7-10 and I can't find any good alternatives with just 6 columns.

**Comment by eladnova** [comment]
Agree. I've ported an existing site from UIKit 2 to 3 and some previous tricky layouts which used 7-10 didn't translate well into the new breakpoints.

**Comment by kiarashi** [comment]
I noticed that uk-width-2-6 and others don't work properly either. 
Guess I've to hold off UIKit 3 for the time being (till these issues have been solved).

**Comment by zzseba78** [comment]
Redundant sizes has been removed, so for example `uk-width-2-6` it's not available, you have to use `uk-width-1-3`. This was made in order to save space, but it affects the workflow in my opinion.
If you need the 10th grid and use less, there is a extended grid size less file that adds 7th 8th 9th 10th and 12th sizes. Of course you can choose that sizes to use, it's great, if you are interested i will search for the link.

**Comment by kiarashi** [comment]
Ah, make sense; will start to change 2-6 to 1-3 and so on.
Regarding the link, yes please share it; I still need a 10- column grid.

**Comment by zzseba78** [comment]
[width-ex.less.zip](https://github.com/uikit/uikit/files/913672/width-ex.less.zip)

**Comment by zzseba78** [comment]
I´m searching the author name

**Comment by AlekVolsk** [comment]
**@zzseba78** please include a link to the solutions page: https://alekvolsk.pw/jed/uikit-grid-width/
i will really appreciate it

**Comment by kiarashi** [comment]
Awesome! Thank you both!

**Comment by zzseba78** [comment]
Done Alek, sorry i didn't find the Gitter chat where you put the source.
Great work!

**Comment by OwenMelbz** [comment]
Incase anybody uses something like npm/yarn to manage their dependencies, they can use https://www.npmjs.com/package/uikit3-extra-widths to add it back in :) Uses Aleks polyfill

**Comment by zzseba78** [comment]
@AlekVolsk seems like your extended width it´s offline.
https://alekvolsk.pw/jed/uikit-grid-width/

**Comment by OwenMelbz** [comment]
@zzseba78 you can use the above link i provided as its a mirror basically

**Comment by AlekVolsk** [comment]
@zzseba78 moved to https://github.com/AlekVolsk/uikit-ex
